modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,93 @@ If you have installed the {gitops-title} Operator in the default namespace, run
+----
+<1> Specifices the name of the `volume` within the Argo CD `ApplicationSet` Controller.
+<2> Specifies the volume type as an `emptyDir`, which creates an empty directory in the pod.
+<3> Specifices the name of the `volumeMounts` within the Argo CD `ApplicationSet` Controller.
+<4> Specifies the path inside the container where the volume will be mounted.
+
+* With this update, the {gitops-shortname} Operator can now gather and export metrics data that is compatible with Prometheus. The data collected by these metrics provide the following important insights into Argo CD instances:
+** The total number of Argo CD instances managed and reconciled.
+** The number of Argo CD instances in various states, such as pending, available, and so on. 
+** The total time for reconciliation loops to complete per instance.
+** The total Resource usage metrics, including CPU, memory, and the number of `go` routines. 
++
+When the {gitops-shortname} Operator is installed in the default `openshift-gitops-operator` namespace, these metrics are automatically available in the {OCP} console. You can access them by navigating to the *Observe* tab and running queries against the monitoring stack. link:https://issues.redhat.com/browse/GITOPS-5674[GITOPS-5674]
+
+* Before this update, users could not configure some replicas for the Argo Rollouts controller by using the `RolloutManager` custom resource (CR). With this update, high availability (HA) support is introduced by adding a new `.spec.ha` field to the `RolloutManager` CR. link:https://issues.redhat.com/browse/GITOPS-5622[GITOPS-5622]
++
+When you enable high availability, the operator automatically sets the number of replicas to 2, the leader election is activated, and default anti-affinity rules are applied to ensure redundancy and reliability.
+
+* With this update, the Argo CD multi-source application functionality in {gitops-shortname} Operator is promoted from Technology Preview (TP) to General Availability (GA). Use this feature to define and manage applications with multiple source repositories. link:https://issues.redhat.com/browse/GITOPS-5511[GITOPS-5511]
+
+* With this update, you can now use Kubernetes TLS-type secrets to configure custom TLS certificates for Routes in the Argo CD CR. Previously, TLS data had to be embedded directly in the Argo CD CR as plain text, which could pose security risks due to the sensitive nature of TLS information.
++
+To enhance security, you must use Kubernetes secrets to manage TLS data. A new field, `.spec.route.tls.externalCertificate`, has been introduced in the Argo CD CR to reference Kubernetes secrets of type `kubernetes.io/tls` containing TLS data. For more information about configuring a route for custom TLS certificate, see examples in link:https://argocd-operator.readthedocs.io/en/latest/usage/routes/#custom-tls-certificates[Custom TLS certificates for Routes]. link:https://issues.redhat.com/browse/GITOPS-5232[GITOPS-5232]
```

Comment:
```suggestion
To enhance security, you must use Kubernetes secrets to manage TLS data. A new field, `.spec.*.route.tls.externalCertificate`, has been introduced in the Argo CD CR to reference Kubernetes secrets of type `kubernetes.io/tls` containing TLS data. For more information about configuring a route for custom TLS certificate, see examples in link:https://argocd-operator.readthedocs.io/en/latest/usage/routes/#custom-tls-certificates[Custom TLS certificates for Routes]. link:https://issues.redhat.com/browse/GITOPS-5232[GITOPS-5232]
```

There are actually multiple fields in ArgoCD CR where `.route` is supported. What do you suggest, should be use wildcard `*` or specify them individually. The fields are `.spec.server.route`, `.spec.applicationSet.webhookServer.Route`, `.spec.prometheus.route` & `.spec.grafana.route`.

Reply:
Using a wildcard causes issues in the build and results in build failure. I would like to list them individually.

Reply:
Fixed.